.circleci/config.yml

```diff
@@ -1,8 +1,7 @@
 version: 2.1
 
 orbs:
-  hmpps: ministryofjustice/hmpps@3.12
-  slack: circleci/slack@4.4.2
+  hmpps: ministryofjustice/hmpps@6
```

Comment:
Big version jump. Not entirely sure about the ramifications of v4 and v5, but I know v6 is safe. I've seen v3 to v5 done in [this PR](https://github.com/ministryofjustice/manage-soc-cases/pull/331)so just chasing up on that in case there's anything to look out for

Reply:
Had a chat with John who did the manage-soc-cases PR. Says it was only the version bump from v3 to v5 that was required as part of the change.